_config.yml

```diff
@@ -72,6 +72,9 @@ heliopy_url: http://heliopy.org
 heliopy_mailinglist: http://groups.google.com/group/plasmapy
 heliopy_riot: https://riot.im/app/#/room/#heliopy:matrix.org
 
+sherpa_repo: https://www.github.com/sherpa/sherpa/
+sherpa_url: http://cxc.cfa.harvard.edu/contrib/sherpa/
```

Comment:
do you maybe have a mailing list or chat channel, too?

Reply:
Currently we don't.